src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -3392,14 +3393,6 @@ namespace winrt::Microsoft::Terminal::Control::implementation
     // -------------------------------- WinRT Events ---------------------------------
     // Winrt events need a method for adding a callback to the event and removing the callback.
     // These macros will define them both for you.
-    DEFINE_EVENT(TermControl, TitleChanged, _titleChangedHandlers, Control::TitleChangedEventArgs);
     DEFINE_EVENT(TermControl, FontSizeChanged, _fontSizeChangedHandlers, Control::FontSizeChangedEventArgs);
     DEFINE_EVENT(TermControl, ScrollPositionChanged, _scrollPositionChangedHandlers, Control::ScrollPositionChangedEventArgs);

```

Comment:
Why can't we get rid of these just like we did with the others?

Reply:
So, those ones got moved in the branch that this actually came from in the first place, so they didn't have a good clean cherry-pick to work with. There's not a _good_ reason other than possibly laziness